src/hls-adapter.js

```diff
@@ -276,7 +289,41 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
       }
     }
   }
-
+  /**
+   * attach media - return the media source to handle the video tag
+   * @public
+   * @param {boolean} playbackEnded don't seek to the last detach point
+   * @returns {void}
+   */
+  attachMediaSource(playbackEnded: ?boolean): void {
+    if (!this._hls) {
+      if (this._videoElement && this._videoElement.src) {
+        Utils.Dom.setAttribute(this._videoElement, 'src', '');
+        Utils.Dom.removeAttribute(this._videoElement, 'src');
```

Comment:
can you explain these 2 lines?

Reply:
Ima on some cases keep empty src, and it made errors

Reply:
Ima on some cases keep empty src, and it made errors

Reply:
so just remove

Reply:
I'm not sure why but if I don't do that it, it's not working..

Reply:
It could happened when IMA doesn't clear the src on ADBREAKEND or didn't finish the clear